tests/check_ssid_wifi.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import os, datetime, pexpect, config, re
+import rootfs_boot
+from devices import board, wan, lan, wlan, prompt
+from lib.installers import install_snmp
+from cbnlib import now_short
+from wifi import WifiScan
+from ssid_set_snmp import SSIDSetSnmp
+from lib.logging import logfile_assert_message
+class scan_ssid_wifi(rootfs_boot.RootFSBootTest):
+    log_to_file = ""
+
+    def WifiTest(self):
+        wan_ip = board.get_interface_ipaddr(board.wan_iface)
+        board.expect(pexpect.TIMEOUT, timeout=60)
+        pass_word = config.wifi_password[0]
+
+        """"Checking wifi connectivity"""
+        wifi_name = ['wifi_2G','wifi_5G']
+        for wifi_device in wifi_name:
+            '''Scanning for SSID'''
+            output = WifiScan(self).runTest()
+            ssid_name = config.board['station']+'SSID'+wifi_device
+
+            '''Matching the unique SSID'''
+            match = re.search(ssid_name,output)
+            logfile_assert_message(self, match!=None,'SSID value check in WLAN container')
+
+            '''Checking for interface status'''
+            self.link_up()
+            
+            '''Generate WPA supplicant file and execute it'''
+            wlan.sendline("rm /etc/"+ssid_name+".conf")
+            wlan.expect(prompt)
+            wlan.sendline("wpa_passphrase "+ssid_name+ " >> /etc/"+ssid_name+".conf")
+            wlan.expect("")
+            wlan.sendline(pass_word)
+            wlan.expect(prompt)
+            wlan.sendline("cat /etc/"+ssid_name+".conf")
+            wlan.expect(prompt)
+            wlan.sendline("wpa_supplicant  -B -Dnl80211 -iwlan1 -c/etc/"+ssid_name+".conf")
+            wlan.expect(prompt)
```

Comment:
This needs to be a function on devices/debian_wifi.py and we call it here- not repeated in each test that needs to do it.